packages/toolpad-app/src/toolpadDataSources/rest/client.tsx

```diff
@@ -279,7 +279,9 @@ function QueryEditor({
   );
 
   const queryScope = {
+    // TODO mark query as @deprecated
```

Comment:
@Janpot same question as above, in addition - if you could give a hint how and if I should provide a different type so it doesn't come up as `Query` type when typed in the binding editor